proto/rill/admin/v1/api.proto

```diff
@@ -1703,39 +1726,104 @@ message RemoveUsergroupMemberUserRequest {
 
 message RemoveUsergroupMemberUserResponse {}
 
+message UserPreferences {
+  optional string time_zone = 1;
+}
+
+message UpdateUserPreferencesRequest {
+  UserPreferences preferences = 1;
+}
+
+message UpdateUserPreferencesResponse {
+  UserPreferences preferences = 1;
+}
+
+message GetUserRequest {
+  string email = 1 [(validate.rules).string.email = true];
+}
+
+message GetUserResponse {
+  User user = 1;
+}
+
 message GetCurrentUserRequest {}
 
 message GetCurrentUserResponse {
   User user = 1;
   UserPreferences preferences = 2;
 }
 
-message GetUserRequest {
+message DeleteUserRequest {
   string email = 1 [(validate.rules).string.email = true];
+  bool superuser_force_access = 2;
```

Comment:
nit: not sure why this and some other unrelated commits are showing up in this PR, they were part of an already merged commit..

Reply:
I added `superuser_force_access` for some of the new APIs, but not the existing ones. I think it shows this way because I moved some of the existing messages to group user-related messages together. Git is tracking the diff in a way that looks like I added it now, but I didn't.